shared/README.md

```diff
@@ -43,28 +42,9 @@ After saving the file set environment using:
 . .env
 ```
 
-Alternative use the variable.tf files in the directories and add the default key on the file as shown on the example below:
-
-``` json
-variable "location" {
-    type = "string"
-    description = "The name of the target location"
-    default = "eastus"
-}
-variable "env" {
-    type = "string",
-    description = "The short name of the target env (i.e. dev, staging, or prod)"
-    defailt = "dev"
-}
-variable "org" {
-    type = "string",
-    description = "The short name of the organization"
-    default = "cse"
-}
-variable "app_name" {
-    type = "string",
-    description = "The short name of the application"
-    default = "cblt"
-}
-
-```
\ No newline at end of file
+Alternatively, use the cluster.tfvars file to set parameter values as shown below:
+
+``` 
+location="esatus"
```

Comment:
Please fix typo

Reply:
@xtophs: Sure will be part PR: https://github.com/Microsoft/entref-appservice-containers/pull/46 since it has to go through the entire review process to push into master.